app/controllers/admin/emergency_messages_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Admin::EmergencyMessagesController < Admin::ApplicationController
+
+  # GET /courts/1/edit
+  def edit
+    @em = EmergencyMessage.find(params[:id])
+  end
+
+  def update
+    @em = EmergencyMessage.find(params[:id])
+
+    respond_to do |format|
+      if @em.update_attributes(params[:emergency_message])
+        purge_all_pages
```

Comment:
Point taken but when that code was removed there must have been an expectation that it would be used at some point. The fact that it hasn't then takes this outside the scope of these sprints, because it needs to be removed from the code base entirely which is not what these sprints are doing.